go/vt/sqlparser/ast.go

```diff
@@ -148,10 +149,16 @@ const (
 	ShareModeStr = " lock in share mode"
 )
 
+// Select.Cache
+const (
+	SQLCacheStr   = "SQL_CACHE "
+	SQLNoCacheStr = "SQL_NO_CACHE "
+)
+
 // Format formats the node.
 func (node *Select) Format(buf *TrackedBuffer) {
-	buf.Myprintf("select %v%s%s%v from %v%v%v%v%v%v%s",
-		node.Comments, node.Distinct, node.Hints, node.SelectExprs,
+	buf.Myprintf("select %s%v%s%s%v from %v%v%v%v%v%v%s",
+		node.Cache, node.Comments, node.Distinct, node.Hints, node.SelectExprs,
```

Comment:
should this go after the comments, since it goes after the comments in the sql.y?